src/Apps/Auction/Routes/ConfirmBid/index.tsx

```diff
@@ -18,21 +20,23 @@ import {
   graphql,
   RelayProp,
 } from "react-relay"
+import { TrackingProp } from "react-tracking"
 import { data as sd } from "sharify"
 import { get } from "Utils/get"
 import createLogger from "Utils/logger"
 
 const logger = createLogger("Apps/Auction/Routes/ConfirmBid")
 
-interface BidProps {
+interface ConfirmBidProps {
   artwork: routes_ConfirmBidQueryResponse["artwork"]
   me: routes_ConfirmBidQueryResponse["me"]
   relay: RelayProp
   location: Location
+  tracking: TrackingProp
 }
 
-export const ConfirmBidRoute: React.FC<BidProps> = props => {
-  const { me, artwork } = props
+export const ConfirmBidRoute: React.FC<ConfirmBidProps> = props => {
+  const { me, artwork, tracking } = props
   const { saleArtwork } = artwork
   const { sale } = saleArtwork
   logger.log({
```

Comment:
@bhoggard Do we think this log is intentional? We should probably remove if not.